subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c

```diff
@@ -159,6 +159,24 @@ int lll_init(void)
 	hal_swi_init();
 
 	/* Connect ISRs */
+#if defined(CONFIG_BT_DISABLE)
+	ARM_IRQ_DIRECT_DYNAMIC_CONNECT_RESCHEDULE(RADIO_IRQn,
+						  CONFIG_BT_CTLR_LLL_PRIO,
+						  IRQ_CONNECT_FLAGS);
+
+	irq_connect_dynamic(RADIO_IRQn, CONFIG_BT_CTLR_LLL_PRIO,
+			    (void (*)(void *))radio_nrf5_isr,
```

Comment:
Use `IRQ_DIRECT_DECLARE` or declare a function of this type based on `CONFIG_BT_DISABLE` instead of casting here